### PR TITLE
refactor(editor): reusable RemovableInputWrapper and AutogrowInput components

### PR DESCRIPTION
**Pull request by hejtful**

### CODE REVIEW

File: packages/editor/src/plugins/fill-in-the-blanks-exercise/components/blank-renderer-input.tsx [hunk 1]
```diff
@@ -21,30 +22,22 @@ export const BlankRendererInput = forwardRef<
...
+      ref={ref}
+      value={text}
+      className={cn(
+        'h-[25px]',
+        isAnswerCorrect && 'border-green-500 focus:outline-green-500',
+        isAnswerCorrect === false && 'border-red-400 focus:outline-red-400'
+      )}
+      onChange={(event) => {
+        setTextUserTypedIntoBlank(event.target.value)
+        onChange?.(event)
```
Comment (elbotho) [inline packages/editor/src/plugins/fill-in-the-blanks-exercise/components/blank-renderer-input.tsx:35]: interesting syntax, did not know that works 👍

File: packages/editor/src/plugins/fill-in-the-blanks-exercise/components/extra-incorrect-answers.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+              <RemovableInputWrapper
+                key={index}
+                tooltipText={blanksExerciseStrings.removeDummyAnswer}
+                onRemoveClick={() => {
+                  handleExtraIncorrectAnswerRemove(index)
+                }}
+              >
+                <AutogrowInput
+                  value={answer}
+                  className="serlo-input-font-reset focus:border-red-400 focus:outline-red-400"
```
Comment (elbotho) [inline packages/editor/src/plugins/fill-in-the-blanks-exercise/components/extra-incorrect-answers.tsx:66]: Not sure if `serlo-input-font-reset` only makes sense here. I feel like it should probably be on all or none of the inputs but did not check.

**Review by elbotho — APPROVED** [review]
thank you!

File: packages/editor/src/plugins/fill-in-the-blanks-exercise/components/extra-incorrect-answers.tsx (hunk 2 above)
Comment (hejtful) [reply packages/editor/src/plugins/fill-in-the-blanks-exercise/components/extra-incorrect-answers.tsx:66]: Before this PR, it was on `AlternativeAnswer` and `ExtraIncorrectAnswer`. `BlankRendererInput` did not have it, not sure why. Trying to add it to `BlankRendererInput` creates styling issues, so I decided that this inconsistency is out of scope for this PR.

**Pull request merged**